packages/jsii-rosetta/lib/commands/transliterate.ts

```diff
@@ -244,7 +200,7 @@ function transliterateType(
       );
       const translation = rosetta.translateSnippet(snippet, language);
```

Comment:
We need the `workingDirectory` here so that we can fixturize properly and find the snippet in `translateSnippet`, correct? Otherwise we would not find the snippet and fail with unknown snippet mode (I think).

Reply:
Fixturization is only necessary for translation, not for lookup

Reply:
But do verify my assumption please

Reply:
https://github.com/aws/jsii/blob/6f6715e29f4eb8a302ab70f762fe4d260528e729/packages/jsii-rosetta/lib/tablets/key.ts#L9-L17

snippet lookup only based on visible source